packages/yara/yara.0.2/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "Anton Kochkov <anton.kochkov@gmail.com>"
+homepage: "https://github.com/XVilka/yara-ocaml"
+bug-reports: "https://github.com/XVilka/yara-ocaml/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/XVilka/yara-ocaml.git"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "dune"
+  "ocaml-migrate-parsetree" {build}
+  "ppx_deriving" {>= "4.2.0"}
+  "core" {>= "v0.9.0" & < "v0.14"}
+  "ctypes" {>= "0.13.0"}
+  "ctypes-foreign"
+]
+depexts: [
+  ["libyara-dev"] {os-family = "debian"}
+  ["libyara-devel"] {os-distribution = "opensuse"}
+  ["epel-release" "yara-devel"] {os-distribution = "centos"}
+  ["yara-devel"] {os-distribution = "fedora"}
+  ["yara"] {os-distribution = "alpine"}
+  ["yara"] {os-distribution = "arch"}
+  ["yara"] {os-distribution = "gentoo"}
+  ["yara"] {os = "macos" & os-distribution = "homebrew"}
+]
```

Comment:
Next time please consider creating a `conf-yara` package as it would reduce the amount of maintenance if the depexts have to change someday.